setup.cfg

```diff
@@ -31,7 +31,7 @@ keywords =
 
 [options]
 python_requires = >=3.6
-packages = ansible_bender
+packages = ansible_bender, ansible_bender.builders, ansible_bender.callback_plugins
```

Comment:
I took look in the python packages docs and found this:
```
packages = find:
```

https://setuptools.readthedocs.io/en/latest/setuptools.html#configuring-setup-using-setup-cfg-files

could we try that instead?

Reply:
That also works.